editor/components/autosave-monitor/index.js

```diff
@@ -15,14 +15,15 @@ import { autosave } from '../../store/actions';
 import {
 	isEditedPostDirty,
 	isEditedPostSaveable,
+	isPostAutosavable,
 } from '../../store/selectors';
 
 export class AutosaveMonitor extends Component {
 	componentDidUpdate( prevProps ) {
-		const { isDirty, isSaveable } = this.props;
-		if ( prevProps.isDirty !== isDirty ||
-				prevProps.isSaveable !== isSaveable ) {
-			this.toggleTimer( isDirty && isSaveable );
+		const { isDirty, isSaveable, isAutosavable } = this.props;
+
+		if ( isDirty && isSaveable && isAutosavable ) {
```

Comment:
ok, that makes sense. i guess this could happen if you made a change then undid it before an autosave would have fired? i'm going to review the current core autosave.js again to see what the expected/current behavior is.